docs/contracts/node-operators-registry.md

```diff
@@ -14,126 +39,306 @@ Returns the rewards distribution proportional to the effective stake for each no
+    uint256 totalExitedValidators,
+    uint256 totalDepositedValidators,
+    uint256 depositableValidatorsCount
+)
+```
+
+#### Returns
+
+| Name                         | Type      | Description                                 |
+| ---------------------------- | --------- | ------------------------------------------- |
+| `totalExitedValidators`      | `uint256` | Total number of exited validators           |
+| `totalDepositedValidators`   | `uint256` | Total number of deposited validators        |
+| `depositableValidatorsCount` | `uint256` | Number of validators which can be deposited |
+
+### getNodeOperatorIsActive()
+
+Returns if the node operator with given id is active.
+
+```sol
+function getNodeOperatorIsActive(uint256 _nodeOperatorId) view returns (bool)
+```
+
+#### Parameters
+
+| Name              | Type      | Description      |
+| ----------------- | --------- | ---------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id |
+
+### getNodeOperatorIds()
+
+Returns up to `_limit` node operator ids starting from the `_offset`.
+
+```sol
+function getNodeOperatorIds(uint256 _offset, uint256 _limit) view
+    returns (uint256[] memory nodeOperatorIds)
+```
+
+#### Parameters
+
+| Name      | Type      | Description                              |
+| --------- | --------- | ---------------------------------------- |
+| `_offset` | `uint256` | Offset of the first element of the range |
+| `_limit`  | `uint256` | Max number of NO ids to return           |
+
+### getNodeOperatorSummary()
+
+Returns some statistics of the staking module.
+
+```sol
+function getNodeOperatorSummary(uint256 _nodeOperatorId) view returns (
+    bool isTargetLimitActive,
+    uint256 targetValidatorsCount,
+    uint256 stuckValidatorsCount,
+    uint256 refundedValidatorsCount,
+    uint256 stuckPenaltyEndTimestamp,
+    uint256 totalExitedValidators,
+    uint256 totalDepositedValidators,
+    uint256 depositableValidatorsCount
+)
+```
+
+#### Returns
+
+| Name                         | Type      | Description                                                                                 |
+| ---------------------------- | --------- | ------------------------------------------------------------------------------------------- |
+| `isTargetLimitActive`        | `bool`    | Is limiting target active validators count for NO enabled                                   |
+| `targetValidatorsCount`      | `uint256` | Used to check how many keys should go to exit, 0 - means all deposited keys would be exited |
+| `stuckValidatorsCount`       | `uint256` | Number of stuck keys from oracle report                                                     |
+| `refundedValidatorsCount`    | `uint256` | Number of refunded keys                                                                     |
+| `stuckPenaltyEndTimestamp`   | `uint256` | Extra penalty time after stuck keys resolved (refunded)                                     |
```

Comment:
resolved **or** refunded (and can even mix them), maybe emphasize

Reply:
I'd rather stuck with `refunded` here and later.